### PR TITLE
Adding spectral entropy to feature table

### PR DESCRIPTION
**Pull request by robinschmid**
Adding two new types and the entropy for the best spectrum in each row

### CODE REVIEW

File: src/main/java/io/github/mzmine/datamodel/features/types/numbers/scores/SpectralEntropyType.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  @NotNull
+  @Override
+  public String getHeaderString() {
+    return "Spectral entropy";
+  }
+
+  @Override
+  public boolean getDefaultVisibility() {
+    return true;
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/datamodel/features/types/numbers/scores/SpectralEntropyType.java:51]: The average user will not care, this should not be enabled by default

File: src/main/java/io/github/mzmine/datamodel/features/types/numbers/scores/NormalizedSpectralEntropyType.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  @NotNull
+  @Override
+  public String getHeaderString() {
+    return "Normalized spectral entropy";
+  }
+
+  @Override
+  public boolean getDefaultVisibility() {
+    return true;
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/datamodel/features/types/numbers/scores/NormalizedSpectralEntropyType.java:51]: The average user will not care, this should not be enabled by default

File: src/main/java/io/github/mzmine/modules/dataprocessing/filter_spectral_entropy_feature/SpectralEntropyModule.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import io.github.mzmine.modules.MZmineModuleCategory;
+import io.github.mzmine.modules.MZmineProcessingModule;
+import io.github.mzmine.parameters.ParameterSet;
+import io.github.mzmine.taskcontrol.Task;
+import io.github.mzmine.util.ExitCode;
+import java.time.Instant;
+import java.util.Collection;
+import org.jetbrains.annotations.NotNull;
+
+public class SpectralEntropyModule implements MZmineProcessingModule {
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/dataprocessing/filter_spectral_entropy_feature/SpectralEntropyModule.java:38]: Do we need a module for it? How about calculating it in the GroupMS2 task?

**Comment by robinschmid** [comment]
Sorry I should have mentioned that.
It was a programming tutorial that I gave. Wanted to upload the video but audio quality was bad

**Pull request closed without merging**